public/viewjs/stocksettings.js

```diff
@@ -51,3 +51,8 @@ $("#stock_default_consume_amount_use_quick_consume_amount").on("click", function
 		$("#stock_default_consume_amount").removeAttr("disabled");
 	}
 });
+
+if (BoolVal(Grocy.UserSettings.use_alternative_actions_on_stock_overview_page))
+{
+	$("#use_alternative_actions_on_stock_overview_page").prop("checked", true);
```

Comment:
New user settings also have to be added to [`config-dist.php`](https://github.com/grocy/grocy/blob/master/config-dist.php), otherwise a "Warning: Undefined array key" will be thrown at many places (please do extensive tests before submitting a PR) unless each user changed this settings once (= each and everyone, existing updated instances and new ones).